DEPLOYMENTS.md

```diff
@@ -0,0 +1,51 @@
+## When do ReportStream deployments take place?
+
+ReportStream production deployments take place on Tuesdays and Thursday around 10am EST.
+
+The lower environments, test and staging, are automatically deployed on every merge to `master` (a.k.a. continuous deployment).
+
+## How is ReportStream deployed?
+
+ReportStream is deployed via [this GitHub Action](.github/workflows/release.yml).
+
+The GitHub Action automatically deploys changes in the following branches to the specified environments:
+
+| Branch | Environment |
+|:--|:--|
+| `master` | test, staging |
+| `production` | production |
+
+Code merged to the above branches always go through a PR review.
+
+The deployment process is zero-downtime with a blue/green deployment process used [via Azure Function deployment slots](https://docs.microsoft.com/en-us/azure/azure-functions/functions-deployment-slots). The deployment process is [documented in PR #1318](https://github.com/CDCgov/prime-reportstream/issues/1318).
+
+Smoke tests are run manually after a deployment, but in the near future the will run automatically before directing traffic to the newly deployed code ([work tracked in #656](https://github.com/CDCgov/prime-reportstream/issues/656)).
```

Comment:
NOTE: you can just do #<issuenumber> and GitHub will render it as a link for you...

```suggestion
Our CI/CD pipeline does not run the Smoke Tests; therefor you *must* run the Smoke tests manually after every deployment. While this will change in the near future (i.e. in the future, they _will_ be included in the pipeline), until that time, *you must run the Smoke tests manually and validate they complete successfully* before directing _any_ traffic to the newly deployed code.
You can track #656 for the status on running the Smoke tests automatically.
```

Reply:
That's what I initially tried, and maybe it doesn't work for PRs, but the issues were not linked.

Reply:
ah, okido, didn't know that... TIL